modules/prebidServerBidAdapter/index.js

```diff
@@ -832,7 +832,8 @@ const OPEN_RTB_PROTOCOL = {
           bidObject.currency = (response.cur) ? response.cur : DEFAULT_S2S_CURRENCY;
 
           // TODO: Remove when prebid-server returns ttl and netRevenue
-          bidObject.ttl = (bid.ttl) ? bid.ttl : DEFAULT_S2S_TTL;
+          const configTtl = _s2sConfig.defaultS2sTtl || DEFAULT_S2S_TTL;
+          bidObject.ttl = (bid.ttl) ? bid.ttl : configTtl;
           bidObject.netRevenue = (bid.netRevenue) ? bid.netRevenue : DEFAULT_S2S_NETREVENUE;
```

Comment:
ttl and netRevenue aren't standard openRTB fields. Please change them to bid.ext.ttl and bid.ext.netRevenue

Reply:
This pr doesnt create these fields; the change you are requesting seems out of scope and breaking.

Reply:
Bummer we didn't catch that before. But you're right.